AI #1332: Add release planning for FOCUS 1.4

jpradocueva opened the pull request:
This PR proposes the timeline for Release FOCUS v1.4

jpradocueva committed:
add release planning for 1_4

jpradocueva committed:
add release planning for 1_4

shawnalpay commented:
Putting this back to draft until @Matt-Cowsert adds 1.4 themes.

jpradocueva committed:
updates from upstream

jpradocueva committed:
update release planning for v1.4 based on 1438 comments

jpradocueva commented:
PR was updated based on the latest discussion on #1332

shawnalpay commented on RELEASE-PLANNING.md:
```diff
@@ -112,6 +112,77 @@ This section outlines the planned release schedule and key milestones for the FO
+    <tr>
+      <td><strong>30-Oct-25 to 9-Apr-26</strong></td>
+      <td>
+          <strong>1.4 Development Starts</strong><br/><ul><li><strong>Thu Oct 30:</strong> (21 weeks + 2-week EOY break)</li></ul>
```
@jpradocueva Each of these entries needs their own row.  Make them rows that follow the parent row.  But it's not readable the way you have it now.

shawnalpay replied:
This looks good now!  Resolving.

Matt-Cowsert committed:
Update RELEASE-PLANNING.md

Added a new row to capture the expected themes for 1.4

jpradocueva committed:
Merge branch 'working_draft' into release_planning_1_4

jpradocueva committed:
Move single line content into multiline (#1506)

Based on the comments received, this PR aims break the content in the
interim column into multiple rows.

---------

Co-authored-by: Shawn Alpay <77511110+shawnalpay@users.noreply.github.com>

shawnalpay commented on RELEASE-PLANNING.md:
```diff
@@ -112,6 +137,100 @@ This section outlines the planned release schedule and key milestones for the FO
+        <td>End-of-year break</td>
+      </tr>
+      <tr>
+        <td><strong>Development Mon Jan 5</strong> </br> (8 weeks)</td>
```
```suggestion
        <td><strong>Mon Jan 5</strong> </br> (8 weeks)</td>
```

shawnalpay commented on RELEASE-PLANNING.md:
@Matt-Cowsert Please update the themes section when you are ready.

Matt-Cowsert replied:
Done!

shawnalpay committed:
Merge branch 'working_draft' into release_planning_1_4

shawnalpay committed:
Merge branch 'working_draft' into release_planning_1_4

jpradocueva committed:
Update RELEASE-PLANNING.md

Good catch!

Co-authored-by: Shawn Alpay <shawn@finops.org>

Matt-Cowsert committed:
Revise RELEASE-PLANNING.md for v1.4 updates

Updated the release planning document for version 1.4 with new features and improvements.

jpradocueva commented:
Approved by the group without objections or abstentions during the October 30 Members' call.
